docs/known-issues/known-issues.md

```diff
@@ -0,0 +1,112 @@
+* [MCLAG leaf with no surviving spine connection will blackhole traffic](#mclag-leaf-with-no-surviving-spine-connection-will-blackhole-traffic)
+
+### Deleting a VPC and creating a new one right away can cause the agent to fail
+
+The issue is due to limitations in SONiC's gNMI interface. In this particular case,
+the deletion and creation of a VPC back-to-back (i.e. using a script or the Kubernetes API)
+can lead to the reuse of the deleted VPC's VNI before the deletion had effect.
+
+#### Diagnosing this issue
+
+The applied generation of the affected agent reported by kubectl will not
+converge to the last desired generation. Additionally, the agent logs on the switch 
+(accessible at `/var/log/agent.log`) will contain an error similar to the following one:
+
+><code>time=2025-03-23T12:26:19.649Z level=ERROR msg=Failed err="failed to run agent: failed to process agent config from k8s: failed to process agent config loaded from k8s: failed to apply actions: GNMI set request failed: gnmi set request failed: rpc error: code = InvalidArgument desc = VNI is already used in VRF VrfVvpc-02"</code>
+
+#### Known workarounds
+
+Deleting the pending VPCs will allow the agent to reconverge. After that, the
+desired VPCs can be safely created.
+
+### Configuration not allowed when port is member of PortChannel
+
+This is another issue related to limitations in SONiC's gNMI interface. Under some circumstances,
+the agent might find itself unable to apply the desired state.
+
+#### Diagnosing the issue
+
+The applied generation of the affected agent reported by kubectl will not
+converge to the last desired generation. Additionally, the agent logs on the switch 
+(accessible at `/var/log/agent.log`) will contain logs similar to the following ones:
+
+><code>time=2025-04-08T14:35:14.555Z level=DEBUG msg=Action idx=4 weight=13 summary="Update Interface Ethernet1 Base" command=update path="/interfaces/interface[name=Ethernet1]"</code>
+
+><code>time=2025-04-08T14:35:14.839Z level=ERROR msg=Failed err="failed to run agent: failed to process agent config from file: failed to apply actions: GNMI set request failed: gnmi set request failed: rpc error: code = InvalidArgument desc = Configuration not allowed when port is member of Portchannel.</code>
+
+#### Known workarounds
+
+Manually setting the interface mentioned immediately before the error log to admin-up,
+i.e. using the sonic-cli on the affected switch. For example, given the logs above, one would
+ssh onto the switch (let's call it `s5248-01`) and do:
+
+```
+admin@s5248-01:~$ sonic-cli
+s5248-01# configure
+s5248-01(config)# interface Ethernet 1
+s5248-01(config-if-Ethernet1)# no shutdown
+```
+
+### VPC local peering can cause the agent to fail if subinterfaces are not supported on the switch
+
+As explained in the [Architecture page](../architecture/fabric.md#vpc-peering), to workaround
+limitations in SONiC, local VPCPeering is implemented over a pair of loopback interfaces.
```

Comment:
It's a bit inaccurate. It's not local VPCPeering implemented that way, it's only when both VPCs or both VPC and Ext are locally attached.

Reply:
You're right, I reworded it to (hopefully) clarify this.